test/clj/nnts2/fixtures.clj

```diff
@@ -1,18 +1,36 @@
 (ns nnts2.fixtures
   (:require [clojure.test :refer :all]
             [clojure.java.jdbc :as sql]
+            [honeysql.helpers :as h]
+            [honeysql-postgres.helpers :as ph]
+            [honeysql.core :as hcore]
             [nnts2.config :as config]
             [nnts2.db :as db]))
 
+(def ^:private test-specs (atom nil))
 
 (defn setup
   [tests]
-  (config/read :test)
-  (db/migrate)
-  (tests))
+  (binding [config/db-spec #(:db-spec @test-specs)]
+    (config/read :test test-specs)
+    (db/migrate)
+    (tests)))
+
+
 
 (defn clear [test]
   (sql/with-db-transaction [db (config/db-spec)]
                            (sql/db-set-rollback-only! db)
                            (binding [config/db-spec (constantly db)]
                              (test))))
+
+
+
+(defn adduser
+  [tests]
```

Comment:
This seems like a helper more than a fixture. Can we not have this as a fixture? Might want to add a different user occasionally.

Reply:
Make sense, I will move it in the future prs though. In this setup, I am totally using it as a fixture.